code/game/machinery/computer/orders/order_items/cook/order_reagents.dm

```diff
@@ -65,3 +65,7 @@
 	name = "Cherry Jelly"
 	item_path = /obj/item/reagent_containers/condiment/cherryjelly
 	cost_per_order = 30
+/datum/orderable_item/reagents/honey
```

Comment:
please put spaces between different order datums, also move this to a monkestation modular folder
```suggestion

/datum/orderable_item/reagents/honey
```